docs/src/components/Demo/Demo.less

```diff
@@ -22,27 +22,25 @@
 
 .options {
   padding: @grid-row-height @grid-gutter-width (@grid-row-height * 2) @grid-gutter-width;
-
-  .aboveSmallDevice({
+  @media @desktop {
```

Comment:
Nice one—now this reads the way we *talk* about this stuff, both devs and designers, which is definitely what we're going for 💯 😎